packages/external-rewards/src/campaigns/LevelMoney.json

```diff
@@ -0,0 +1,21 @@
+{
+  "campaignName": "",
```

Comment:
```suggestion
  "campaignName": "LevelMoney Liquidity",
```

Reply:
we did not used campaign name in any campaigns so far, not a problem to add but we should add campaign names accross all campaigns